classes/migration/install/CommonMigration.php

```diff
@@ -96,8 +96,8 @@ public function up(): void
                 break;
         }
 
-        // Locale-specific user data
         Schema::create('user_settings', function (Blueprint $table) {
+            $table->comment('Localized data about users, like their name and affiliation.');
```

Comment:
Not all `_settings` entries are localized, e.g. the ORCiD plugin uses `user_settings` to store ORCiD IDs for user accounts without requiring its own table or schema extension.